Modules/_ctypes/_ctypes.c

```diff
@@ -2596,10 +2596,45 @@ static PyMemberDef PyCData_members[] = {
     { NULL },
 };
 
+/*
+  Get the StgDictObject corresponding to a single item of a multidimensional
+  array.
+  Takes and returns a borrowed reference.
+*/
+static StgDictObject *PyCData_item_stgdict(StgDictObject *dict)
```

Comment:
`PyCData_item_stgdict(StgDictObject *dict)` should be on its own line to comply with [PEP 7](https://www.python.org/dev/peps/pep-0007/#code-lay-out).

Reply:
This matches the style of `PyCData_NewGetBuffer` below. Shall I change both?

Reply:
Yes, since most of the functions in that file follow PEP 7.